### PR TITLE
LUT-26977: Add a validator for date field

### PR DESCRIPTION
**Pull request by husel-t**
Number of minimum elapsed days 
https://dev.lutece.paris.fr/bugtracker/issues/26977

### CODE REVIEW

File: src/java/fr/paris/lutece/plugins/forms/validation/AbstractDayDistanceValidator.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+    @Override
+    public String getDisplayHtml( Control control )
+    {
+        Map<String, Object> model = new HashMap<>( );
+        model.put( FormsConstants.PARAMETER_CONTROL_VALUE, control.getValue( ) );
+        model.put( MARK_LABEL, getLabelKey( ) );
+        model.put( MARK_HELP, getHelpKey( ) );
+
+        HtmlTemplate htmlTemplateQuestion = AppTemplateService.getTemplate( TEMPLATE_DISPLAY_HTML, I18nService.getDefaultLocale( ), model );
```
Comment (seboo) [inline src/java/fr/paris/lutece/plugins/forms/validation/AbstractDayDistanceValidator.java:49]: the current user may have chosen another locale...

Comment (seboo) [reply src/java/fr/paris/lutece/plugins/forms/validation/AbstractDayDistanceValidator.java:49]: ok, the related PR is merged, could you use this new parameter ?

Comment (husel-t) [reply src/java/fr/paris/lutece/plugins/forms/validation/AbstractDayDistanceValidator.java]: new parameter added 👍

**Comment by seboo** [comment]
Please resolve conflicts.

Are the 23 commits of this PR related to  LUT-26977 ?

**Pull request merged**